calcom/cal.com
# Change Stripe `<CardElement />` to `<PaymentElement />`

## What does this PR do?



This PR makes changes to the booking payment page using Stripe. The `<CardElement />` was not rendering properly on mobile so we changed to using `<PaymentElement />` which is recommended by Stripe.

**Before**
![image](https://user-images.githubusercontent.com/65426560/232080728-bb9ab41a-648a-4f0f-b668-a76adfc04167.png)
Hard for a user on mobile to edit their card details

**With holds**
<img width="368" alt="image" src="https://user-images.githubusercontent.com/65426560/232080110-2b7307ab-2d88-4567-83d9-3becb0c144c2.png">

**Dark mode**
<img width="363" alt="image" src="https://user-images.githubusercontent.com/65426560/232080664-fc2c627f-20e3-42fc-85fe-21a7ceb25262.png">




Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change



- Bug fix (non-breaking change which fixes an issue)
- Chore (refactoring code, technical debt, workflow improvements)


## How should this be tested?



- Create an event type with the Stripe app for payment
- For both payment on booking & no-show fee create a booking
- On the booking payment page,  view the page on desktop & mobile size
    - The booking page should look like the screenshots above
    - The payment should either go through or the payment information should be stored on Stripe depending on the event type settings
- On the booking success page, if the event type does not require confirmation then it should show success

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
